js_modules/dagit/packages/core/src/runs/PipelineRunLogsSubscription.tsx

```diff
@@ -0,0 +1,197 @@
+import {gql} from '@apollo/client';
+
+import {METADATA_ENTRY_FRAGMENT} from '../app/MetadataEntryFragment';
+import {PYTHON_ERROR_FRAGMENT} from '../app/PythonErrorFragment';
+
+const RUN_METADATA_PROVIDER_MESSAGE_FRAGMENT = gql`
+  fragment RunMetadataProviderMessageFragment on DagsterRunEvent {
+    __typename
+    ... on MessageEvent {
+      message
+      timestamp
+      stepKey
+    }
+    ... on MarkerEvent {
+      markerStart
+      markerEnd
+    }
+    ... on ObjectStoreOperationEvent {
+      operationResult {
+        op
+        metadataEntries {
+          ...MetadataEntryFragment
+        }
+      }
+    }
+    ... on LogsCapturedEvent {
+      logKey
+      stepKeys
+      pid
+    }
+  }
+  ${METADATA_ENTRY_FRAGMENT}
+`;
```

Comment:
They exist elsewhere but I redefined them here so that they'd be packaged separately with the worker. Otherwise the worker bundle would likely include unused javascript from having a bigger bundle. I'll fix it like the original PR though so theres only 1 definition